### ISSUE TITLE
Implement statusnet/groups/list API

### ISSUE THREAD

**Issue by Rudloff** [friendica/friendica#4135]
It seems we don't support GNU Social groups yet but I guess we could at least return Friendica forums.

**Comment by annando** [comment]
I don't think that it does make sense to do something for the Statusnet style API any more.

**Comment by tobiasd** [comment]
But it is in the Mastodon compatible API right?

**Comment by annando** [comment]
Groups in AP are done in a completely other way.
